DokanStore/DokanStore/XIB/CustomView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomView.swift
+//  DokanStore
+//
+//  Created by heba isaa on 18/06/2022.
+//
+
+import UIKit
+
+class CustomView: UIViewFromNib {
```

Comment:
The class should have a more specific name

Reply:
this is a example just for a UI Method